.github/workflows/tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run Unit Tests
```

Comment:
Actually, pytest is included in the existing [static-checks](https://github.com/maxsvetlik/balatro-gym/blob/94236c308660303d35a6914903eccff49da36cec/.github/workflows/static-checks.yml#L28) workflow.
It's not really a static check, but was initially convenient and I think it makes sense to break it out. Can you remove the existing pytest trigger from static-checks?

Reply:
Oh I missed that! In my previous PR the unit tests were failing locally, but showed that it passed on GH. I'll purposely push a breaking test just to verify if now it catches failing tests :eyes:

Reply:
Okay, I understand why it wasn't caught by previous checks. I'm on python 3.10 (since I saw the requirements were >=3.10) and the enum deep copy issue only occurred on python 3.10 (https://github.com/maxsvetlik/balatro-gym/actions/runs/14999699037/job/42143053742?pr=6), but the tests weren't running on this version of python, so I added that here.